core/common/modules/finder/assets/js/component.js

```diff
@@ -41,6 +41,35 @@ export default class Component extends ComponentModalBase {
 		const modalCommands = super.defaultCommands();
 
 		return {
+			'navigate/down': () => {
+				elementorCommon.helpers.softDeprecated(
+					"$e.run( 'finder/navigate/down' )",
+					'3.0.0',
+					"$e.run( 'finder/navigate-down' )"
+				);
+
+				$e.run( 'finder/navigate-down' );
+			},
+			'navigate/up': () => {
+				elementorCommon.helpers.softDeprecated(
+					"$e.run( 'finder/navigate/up' )",
+					'3.0.0',
+					"$e.run( 'finder/navigate-up' )"
+				);
+
+				$e.run( 'finder/navigate-up' );
+			},
+			'navigate/select': ( event ) => {
+				elementorCommon.helpers.softDeprecated(
+					"$e.run( 'finder/navigate/select', event )",
+					'3.0.0',
+					"$e.run( 'finder/navigate-select', event )"
+				);
+
+				// TODO: Fix $e.shortcuts use args. ( args.event ).
```

Comment:
Why?

Reply:
@matipojo

Reply:
It's backwards compatibility for old command like `navigate/select` that now it's `navigate-select`.